hw_recogs.ipynb

```diff
@@ -16,7 +16,7 @@
    "outputs": [],
    "source": [
     "__author__ = \"Christopher Potts and Zhengxuan Wu\"\n",
-    "__version__ = \"CS224u, Stanford, Spring 2023\""
+    "__version__ = \"CS224u, Stanford, Spring 2024\""
    ]
   },
   {
```

Comment:
From "you try some in-context learning with DSP." to "you try some in-context learning with DSPy." in overview section.

Reply:
Thanks!